lib/phx_tools_web/live/phx_tools_live/instruction_components.ex

```diff
@@ -20,13 +20,13 @@ defmodule PhxToolsWeb.InstructionComponents do
           <h1 class="text-white text-center lg:text-2xl md:text-[22px] sm:text-xl lg:mb-[5%] md:mb-[2%] sm:mb-[2%]">
             <%= render_slot(@header) %>
           </h1>
-          <div class="h-full w-full border-4 shadow-[#28177E] shadow-md border-[#211666] rounded-md lg:mx-[2%] pb-2">
+          <div class="h-full w-full border-4 hover:border-2 shadow-[#28177E] shadow-md border-[#211666] hover:border-indigo-450 rounded-md lg:mx-[2%] pb-2">
```

Comment:
@almirsarajcic this is an optional addition. I don't know how the design looks like.

Reply:
I checked the design, and this addition is not present in the design. I still think it's an improvement worth keeping. What do you think, @almirsarajcic.

Reply:
I had a call with Farah, and we agreed the hover styling is not necessary. I have reverted the commit.